--- uspell/src/lookup2.cpp.orig	2003-07-17 01:05:59.000000000 +0200
+++ uspell/src/lookup2.cpp	2006-02-19 12:04:26.246857250 +0100
@@ -206,7 +206,7 @@
    c = initval;           /* the previous hash value */
 
    /*---------------------------------------- handle most of the key */
-   if (((ub4)k)&3)
+   if (reinterpret_cast<unsigned long>(k) & 3)
    {
       while (len >= 12)    /* unaligned */
       {
--- uspell/src/transcribe.cpp.orig	2003-07-17 01:06:00.000000000 +0200
+++ uspell/src/transcribe.cpp	2006-02-19 12:40:55.003646000 +0100
@@ -143,7 +143,8 @@
 			aftSource = foreSource;
 			current = &transcribeStart;
 		} else { // no replacement; output one byte and rescan
-			*(reinterpret_cast<char *>(destPtr))++ = *aftSource++;
+			*reinterpret_cast<char *>(destPtr) = *aftSource++;
+			destPtr = reinterpret_cast<wide_t*>(reinterpret_cast<char *>(destPtr) + 1);
 			foreSource = aftSource;
 			current = &transcribeStart;
 		}
--- uspell/src/uspell.h.orig	2003-07-24 02:20:04.000000000 +0200
+++ uspell/src/uspell.h	2006-02-19 12:41:16.877013000 +0100
@@ -136,7 +136,7 @@
 			const wide_t *target, const int targetLength);
 		int wordDiff(const wide_t *string1, const int string1Length,
 			const wide_t *string2, const int string2Length);
-		void uSpell::acceptGoodWord(const utf8_t *buf, int wordPosition,
+		void acceptGoodWord(const utf8_t *buf, int wordPosition,
 			int fileNumber);
 }; // class uSpell
 
