diff -urN aa/uspell/src/transcribe.cpp uspell/src/transcribe.cpp
--- aa/uspell/src/transcribe.cpp	2003-07-17 01:06:00.000000000 +0200
+++ uspell/src/transcribe.cpp	2004-06-29 01:27:01.282138288 +0200
@@ -143,7 +143,7 @@
 			aftSource = foreSource;
 			current = &transcribeStart;
 		} else { // no replacement; output one byte and rescan
-			*(reinterpret_cast<char *>(destPtr))++ = *aftSource++;
+			*(reinterpret_cast<char *&>(destPtr))++ = *aftSource++;
 			foreSource = aftSource;
 			current = &transcribeStart;
 		}
